apps/integration/integration/ingests/index.py

```diff
@@ -1,11 +1,12 @@
 import pytest
 from integration.ingests.crawler import HttpCrawlerIndex
+from integration.ingests.jupyter import JupyterIndex
 
+INGEST_PROFILES = ["crawler-http-one", "crawler-http-all", "jupyter-default-prep", "jupyter-dev-example"]
```

Comment:
Do we care about sycamore_demo?

Reply:
If by that you mean `jupyter/sycamore_demo.ipynb`, then it isn't in the container, so no. If there's some other sycamore_demo then I don't know what it is